pyproject.toml

```diff
@@ -27,6 +27,10 @@ dynamic = ["version"]
 "Homepage" = "https://github.com/peterridolfi/Pyhf-to-Combine-converter"
 "Bug Tracker" = "https://github.com/peterridolfi/Pyhf-to-Combine-converter/issues"
 
+[project.scripts]
+pyhf-to-combine = "pyhf_combine_converter.pyhf_convert_to_datacard:main"
+combine-to-pyhf = "pyhf_combine_converter.pyhf_converted_from_datacard:main"
```

Comment:
@alexander-held @peterridolfi you may have discussed the CLI API before and have strong feeling on this, but this seemed clear and short to me.

Reply:
I think this is a good suggestion, it is what I would intuitively want to use on the command line.